### ISSUE TITLE
关于复现精度的问题，无法达到您说的精度，可以公布一下您达到最好精度时的随机种子，谢谢

### ISSUE THREAD

**Issue by xiaobaitu123344** [ma-xu/pointMLP-pytorch#68]

**Comment by ma-xu** [comment]
@xiaobaitu123344 
You can find all hyper-parameters (including reandom seeds) in the logs, e.g., https://web.northeastern.edu/smilelab/xuma/pointMLP/checkpoints/fixstd/modelnet40/pointMLP-20220209053148-404/out.txt

Also, please refer https://github.com/ma-xu/pointMLP-pytorch/issues/67 for random seed issue, and https://github.com/ma-xu/pointMLP-pytorch/issues/1#issuecomment-1032642340 for reproducibility issue.
